admin-ui/src/components/token-dialog.tsx

```
@@ -112,9 +133,15 @@ export function TokenDialog(): React.JSX.Element {
             type="password"
             placeholder={t("tokenDialog.inputPlaceholder")}
             value={draft}
-            onChange={(e) => setDraft(e.target.value)}
+            onChange={(e) => {
+              setDraft(e.target.value)
+              if (saveError) setSaveError(null)
+            }}
             autoComplete="off"
           />
+          {saveError ? (
```

ESLint

[error] Delete `·(`

(prettier/prettier)